app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -379,6 +379,16 @@ fun openLink(url: String, ctx: Context, useCustomTab: Boolean) {
     if (useCustomTab) {
         val intent = CustomTabsIntent.Builder()
             .build()
+
```

Comment:
Why does this only apply to custom tabs?

Reply:
Fixed